cmd/osv-scanner/main.go

```diff
@@ -84,6 +85,42 @@ func run(args []string, stdout, stderr io.Writer) int {
 	return 0
 }
 
+func getCustomHelpTemplate() string {
+	return `
+NAME:
+	{{.Name}} - {{.Usage}}
+
+USAGE:
+	{{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}{{if .Commands}} command [command options]{{end}}
+
+EXAMPLES:
+
+	# Scan a source directory
+	$ {{.HelpName}} scan source -r <source_directory>
+
+	# Scan a container image:
+	$ {{.HelpName}} scan image <image_name>
+
+	# Scan a local image archive (e.g. a tar file) and generate HTML output:
+	$ {{.HelpName}} scan image --serve --archive <image_name.tar>
+
+	# Fix vulnerabilities in a manifest file and lockfile (non-interactive mode):
+	$ {{.HelpName}} fix --non-interactive -M <manifest_file> -L <lockfile>
+
+	For full usage details, please refer to the help command of each subcommand (e.g. {{.HelpName}} scan --help).
+
+VERSION:
+	{{.Version}}
+
+COMMANDS:
+{{range .Commands}}{{if not .HideHelp}}  {{join .Names ", "}}{{ "\t"}}{{.Usage}}{{ "\n" }}{{end}}{{end}}
```

Comment:
I think the "update" command is meant to be hidden for now. I see in your screenshot that it's currently being shown?

Reply:
I didn't notice that, thanks for catching it.

updated code to remove hidden commands from help text